crates/sui-core/src/authority/consensus_quarantine.rs

```diff
@@ -495,6 +512,9 @@ pub(crate) struct ConsensusOutputQuarantine {
 
     processed_consensus_messages: RefCountedHashMap<SequencedConsensusTransactionKey, ()>,
 
+    // Owned object locks acquired post-consensus
+    owned_object_locks: RefCountedHashMap<ObjectRef, LockDetails>,
```

Comment:
Afaict we never read this, which means we won't be able to detect conflicts correctly.